pkg/base/base.go

```diff
@@ -43,16 +44,19 @@ func init() {
 	troubleshootscheme.AddToScheme(scheme.Scheme)
 }
 
-func GetGVKWithNameHash(content []byte) []byte {
+func GetGVKWithNameAndNs(content []byte) string {
 	o := OverlySimpleGVK{}
 
 	if err := yaml.Unmarshal(content, &o); err != nil {
-		return nil
+		return ""
+	}
+
+	namespace := os.Getenv("POD_NAMESPACE")
```

Comment:
base.Base has a Namespace property. I dont think this will work with helm charts

Reply:
isn't this after the helm template code, though? So that'd already be written into the yaml

Reply:
https://github.com/replicatedhq/kots/blob/master/pkg/base/write.go#L103-L105